test/fixtures/semantic-html5-scenarios/section-with-id.adoc

```diff
@@ -0,0 +1,4 @@
+[#tigers-subspecies]
```

Comment:
If a fixture is going to start at a section level below level 1, it should be marked as a fragment document.

Reply:
Are you suggesting that we use a naming convention (i.e., for instance a prefix `fragment-section-id.adoc`) or that we use two distinct set of fixtures in order to convert fragment documents or complete documents?

Reply:
I'm saying to avoid warnings, a document which doesn't start with a top-level section should define the `fragment` attribute at the top of the document. See https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes-reference/#section-title-and-table-of-contents-attributes (Technically, it's not required, but considered a good practice IMO).

Reply:
Oh I wasn't aware of this attribute, noted!